test/test_unit_conjur_resource.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from conjur.resource import Resource
+
+class ConjurResourceDataTest(unittest.TestCase):
+
+    def test_full_id_returns_pair_properly(self):
+        mock_resource = Resource(type_="sometype", name="somename")
+        resource_pair = mock_resource.full_id()
+
+        assert resource_pair == "sometype:somename"
+
+    def test_host_input_data_rotate_api_key_is_printed_as_dict_properly(self):
+        EXPECTED_REP_OBJECT="'type': 'sometype', 'name': 'somename'"
```

Comment:
why this is UPPERCASE const ? cant it be regular variable?

Reply:
I personally like it uppercase so it stands out. I don't want to push it up to the top of the file because there are many EXPECTED_REP_OBJECT for different tests